src/main/scala/is/hail/variant/VariantDataset.scala

```diff
@@ -1122,15 +1122,15 @@ class VariantDatasetFunctions(private val vds: VariantSampleMatrix[Genotype]) ex
 
           vEC.setAll(v, va)
           vf().foreach { x =>
-            ab += x.orNull
+            ab += x
```

Comment:
This is an unsafe use of ArrayBuilder here.

We know the result size (vf + gf) so we may as well just allocate an array and arraycopy results